content/docs/reference/tfjob/v1beta2/tensorflow.md

```diff
@@ -0,0 +1,226 @@
++++
+title = "TFJob TensorFlow"
+description = "Reference documentation for TFJob"
+weight = 100
++++
+<p>Packages:</p>
+<ul>
+<li>
+<a href="#kubeflow.org">kubeflow.org</a>
+</li>
+</ul>
+<h2 id="kubeflow.org">kubeflow.org</h2>
+<p>
+<p>Package v1beta2 is the v1beta2 version of the API.</p>
+</p>
+Resource Types:
+<ul><li>
+<a href="#TFJob">TFJob</a>
+</li></ul>
+<h3 id="TFJob">TFJob
+</h3>
+<p>
+<p>TFJob represents the configuration of signal TFJob</p>
+</p>
+<table>
+<thead>
+<tr>
+<th>Field</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<code>apiVersion</code></br>
+string</td>
+<td>
+<code>
+kubeflow.org/v1beta2
+</code>
+</td>
+</tr>
+<tr>
+<td>
+<code>kind</code></br>
+string
+</td>
+<td><code>TFJob</code></td>
+</tr>
+<tr>
+<td>
+<code>metadata</code></br>
+<em>
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#objectmeta-v1-meta">
+Kubernetes meta/v1.ObjectMeta
+</a>
+</em>
+</td>
+<td>
+<p>Standard object&rsquo;s metadata.</p>
+Refer to the Kubernetes API documentation for the fields of the
+<code>metadata</code> field.
+</td>
+</tr>
+<tr>
+<td>
+<code>spec</code></br>
+<em>
+<a href="#TFJobSpec">
+TFJobSpec
+</a>
+</em>
+</td>
+<td>
+<p>Specification of the desired behavior of the TFJob.</p>
+<br/>
+<br/>
+<table>
+<tr>
+<td>
+<code>cleanPodPolicy</code></br>
+<em>
+<a href="https://kubeflow.org/docs/references/tfjob/v1beta2/common/#CleanPodPolicy">
```

Comment:
This link is broken - It has "references" instead of "reference" in the path, and it's also an absolute link. Can you change it to relative?